subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -145,15 +149,20 @@ static int init_block_ctx(const uint8_t *token, uint8_t tkl, struct lwm2m_block_
 	return 0;
 }
 
-static int get_block_ctx(const uint8_t *token, uint8_t tkl, struct lwm2m_block_context **ctx)
+static int get_block_ctx(const struct lwm2m_obj_path *path, struct lwm2m_block_context **ctx)
 {
 	int i;
 
+	if (!path) {
+		LOG_ERR("Null block ctx path");
+		return -EFAULT;
+	}
+
 	*ctx = NULL;
 
 	for (i = 0; i < NUM_BLOCK1_CONTEXT; i++) {
-		if (block1_contexts[i].tkl == tkl &&
-		    memcmp(token, block1_contexts[i].token, tkl) == 0) {
+		if (memcmp(path, &lwm2m_block1_context()[i].path,
+		sizeof(struct lwm2m_obj_path)) == 0) {
```

Comment:
This still looks wrongly aligned (please align to the opening bracket):
```suggestion
		if (memcmp(path, &lwm2m_block1_context()[i].path,
			   sizeof(struct lwm2m_obj_path)) == 0) {
```

Reply:
Done